frontend/src/customer/specification/Specification.scss

```diff
@@ -0,0 +1,286 @@
+@import url('https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css');
+
+/* Kontener bazowy */
+.base{
```

Comment:
Podobnie jak przy klasach w JS, jeśli jest potrzebny komentarz przy klasach w CSS to znaczy, że są źle nazwane.